demisto_sdk/commands/pre_commit/hooks/hook.py

```diff
@@ -59,6 +65,11 @@ def _set_files_on_hook(
         if should_filter:
             files_to_run_on_hook = self.filter_files_matching_hook_config(files)
         hook["files"] = join_files(files_to_run_on_hook)
+        if use_args:
```

Comment:
whats this do?

Reply:
because we run the script not in content working directory we will have to provide the filepaths in the `args` and mark `pass_filenames` to false (because the filenames are relative to content)